packages/@aws-cdk/aws-codecommit/lib/pipeline-action.ts

```diff
@@ -31,6 +28,16 @@ export interface PipelineSourceProps extends codepipeline.CommonActionProps {
     pollForSourceChanges?: boolean;
 }
 
+/**
+ * Construction properties of the {@link PipelineSource CodeCommit source CodePipeline Action}.
```

Comment:
FYI - the `{@link}`isn't going to render properly in our Sphinx documentation (not that this isn't something we should fix :D)

Reply:
This is actually just a moved comment, so it doesn't bother me that much... yet ;)